Objects/obmalloc.c

```diff
@@ -1533,31 +1534,60 @@ _PyMem_DebugRawRealloc(void *ctx, void *p, size_t nbytes)
     }
     total = nbytes + 4*SST;
 
+    tail = q + original_nbytes;
+    /* Mark the header, the trailer, ERASED_SIZE bytes at the begin and
+       ERASED_SIZE bytes at the end as dead and save the copy of erased bytes.
+     */
+    if (original_nbytes <= 2*ERASED_SIZE) {
```

Comment:
I would prefer to use sizeof(save) in the test.

Reply:
Do you prefer to use `sizeof(save)/2` instead of `ERASED_SIZE`?